lrlex/src/lib/ctbuilder.rs

```diff
@@ -1183,99 +1183,201 @@ impl CTLexer {
     }
 }
 
-/// Create a Rust module named `mod_name` that can be imported with
-/// [`lrlex_mod!(mod_name)`](crate::lrlex_mod). The module contains one `const` `StorageT` per
-/// token in `token_map`, with the token prefixed by `T_`. In addition, it will
-/// contain an array of all token IDs `TOK_IDS`.
+/// Exports all token IDs used by a parser as a separate Rust module.
+///
+/// This builder will create a Rust module named `mod_name`
+/// that can be imported with [`lrlex_mod!(mod_name)`](crate::lrlex_mod).
+/// The module will contain one `const` `StorageT` per token in `token_map`,
+/// with the token prefixed by `T_`. In addition, it will contain
+/// an array of all token IDs `TOK_IDS`.
 ///
-/// For example with `StorageT` `u8`, `mod_name` `x`, and `token_map`
+/// For example, if `StorageT` is `u8`, `mod_name` is `x`, and `token_map` is
 /// `HashMap{"ID": 0, "INT": 1}` the generated module will look roughly as follows:
 ///
 /// ```rust,ignore
 /// mod x {
 ///   pub const T_ID: u8 = 0;
 ///   pub const T_INT: u8 = 1;
-///   pub const TOK_IDS: &[u8] = &[T_ID, T_INT];
+///   pub const TOK_IDS: &[u8] = &[0, 1];
```

Comment:
Rereading this, I realise that the constants `0` and `1` are more accurate, but less helpful: I briefly had to remind myself what they refer to! I think it would be better to keep the variable names in here. *Technically* (I think...) they both lead to the same observable values because of `const`.